CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # `wasm-bindgen` Change Log
 --------------------------------------------------------------------------------
 
+## [0.2.105](https://github.com/wasm-bindgen/wasm-bindgen/compare/0.2.104...0.2.105)
+
+### Fixed
+
+* Fixed multithreading JS output for targets `bundler`, `deno` and `module`.
```

Comment:
I was actually under the impression until now that they weren't exactly broken, just unsupported. But looking at the reference output its clearly broken.

Reply:
Yup, they would literally fail trying to import a non-existing module.